include/mata/utils/utils.hh

```diff
@@ -112,6 +112,16 @@ extern const std::string g_GIT_SHA1;
  */
 namespace utils {
 
+inline std::string replace_all(const std::string& input, const std::string& match, const std::string& replace) {
```

Comment:
This should have a documentation. Also, maybe `needle` is more common than `match`?